Tests/SonataCoreBundleTest.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\CoreBundle\Tests;
+
+use Sonata\CoreBundle\DependencyInjection\Compiler\AdapterCompilerPass;
+use Sonata\CoreBundle\DependencyInjection\Compiler\FormFactoryCompilerPass;
+use Sonata\CoreBundle\DependencyInjection\Compiler\StatusRendererCompilerPass;
+use Sonata\CoreBundle\Form\FormHelper;
+use Sonata\CoreBundle\SonataCoreBundle;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+
+/**
+ * Test for SonataCoreBundle.
+ *
+ * @author Ahmet Akbana <ahmetakbana@gmail.com>
+ */
+final class SonataCoreBundleTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @group core-bundle
+     * @covers Sonata\CoreBundle\SonataCoreBundle::build
+     */
+    public function testBuild()
+    {
+        $containerBuilder = $this->getMock('Symfony\Component\DependencyInjection\ContainerBuilder');
+
+        $containerBuilder->expects($this->exactly(3))
+            ->method('addCompilerPass')
+            ->will($this->returnCallback(function (CompilerPassInterface $pass) {
+                if ($pass instanceof StatusRendererCompilerPass) {
+                    return;
+                }
+
+                if ($pass instanceof AdapterCompilerPass) {
+                    return;
+                }
+
+                if ($pass instanceof FormFactoryCompilerPass) {
+                    return;
+                }
+
+                $this->fail(sprintf('Compiler pass is not one of the expected types. Expects "Sonata\AdminBundle\DependencyInjection\Compiler\StatusRendererCompilerPass", "Sonata\AdminBundle\DependencyInjection\Compiler\AdapterCompilerPass" or "Sonata\AdminBundle\DependencyInjection\Compiler\FormFactoryCompilerPass", but got "%s".', get_class($pass)));
+            }));
```

Comment:
AFAIK, the discussion is not over: https://github.com/sonata-project/dev-kit/issues/89 :-)